tox.ini

```diff
@@ -15,12 +15,15 @@ setenv =
 extras = tests
 commands =
     pip freeze --all --no-input
+    python -c 'from astropy.utils.iers import IERS_Auto; IERS_Auto.open()'
```

Comment:
```suggestion
    # pre-cache the IERS file for astropy to prevent downloads with remote_data off
    python -c 'from astropy.utils.iers import IERS_Auto; IERS_Auto.open()'
```